tools/syz-trace2syz/config/unsupported_calls.go

```diff
@@ -0,0 +1,45 @@
+package config
+
+var (
+	// ShouldSkip lists system calls that we should skip when parsing
+	// Some of these are unsupported or not worth executing
+	ShouldSkip = map[string]bool{
+		"brk":                    true,
```

Comment:
I would like to see exact reasons we ignore these calls, because in future we may want to enable more and it would be useful to know if there is some fundamental reason why it's ignored or if support is not implemented yet.
No need to comment each syscall independently, but at least do some groups of syscalls, e.g.:
```
// These don't give any interesting coverage:
getcpu
getcwd
// These something else...:
...
...
```

Also I see some syscalls here that are not even present in main descriptions (notably, "reboot" and ""). I think the first step should be to check target.SyscallMap and if it's not present there, then skip it. No need to duplicate such calls here (we never want to do reboot).

But I would expect that things like mprotect/mremap/futex we want to support in future.

Reply:
Yeah I agree. I'm thinking we could have a config file similar to mgr.cfg where you can specify "Disable Calls" or "Enable Calls". Users may have a set of traces and want to only generate programs that contain "open, ioctl, write" or only exclude a few like "execve, clone, etc".

Reply:
Let's get the basic version working first ;)

Reply:
Sure!